README.md

```diff
@@ -11,4 +11,4 @@ NoMoreDomains checks the URL of every site you visit to see if it is one of over
 
 NoMoreDomains is only in early development, we plan to add a redirect to a branded page eventually and create a website for NoMoreDomains. As NoMoreDomains is open source we rely solely on contributions to build this chrome extension. If you wish to make a contribution please visit the [issues](https://github.com/immattdavison/NoMoreDomains/issues) and find issues that need assistance or create one of your own!
 
-When making contributions please consider and follow our [Contributing Guidelines](https://github.com/immattdavison/NoMoreDomains/blob/master/contributing.md).
\ No newline at end of file
+When making contributions please consider and follow our [Contributing Guidelines](https://github.com/immattdavison/NoMoreDomains/blob/master/contributing.md).
```

Comment:
unsure of the intension of this change?

Reply:
I formatted the code via prettier and it tried to remove any whitespace/newlines.